aws/data_source_aws_ecs_container_definition.go

```diff
@@ -93,10 +98,20 @@ func dataSourceAwsEcsContainerDefinitionRead(d *schema.ResourceData, meta interf
 		d.Set("docker_labels", aws.StringValueMap(def.DockerLabels))
 
 		var environment = map[string]string{}
-		for _, keyValuePair := range def.Environment {
-			environment[aws.StringValue(keyValuePair.Name)] = aws.StringValue(keyValuePair.Value)
+		for _, envKeyValuePair := range def.Environment {
+			environment[aws.StringValue(envKeyValuePair.Name)] = aws.StringValue(envKeyValuePair.Value)
 		}
 		d.Set("environment", environment)
+
+		var portMappings = make([]map[string]string, 0)
```

Comment:
Can you change this so that it has a fixed and expected size, based on def.PortMappings?
As in: `var portMappings = make([]map[string]string, len(def.PortMappings), len(def.PortMappings))`